www/index.html

```diff
@@ -78,7 +78,6 @@
 </script>
 
 <script src="boot.js"></script>
-<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch,Promise,Array.from,Array.prototype.find,Array.prototype.includes,String.prototype.startsWith,String.prototype.includes,Math.sign,Intl.~locale.en,Intl.~locale.id,Intl.~locale.ru,Intl.~locale.fr&unknown=polyfill"></script>
```

Comment:
I don't think iPod is that important to bloat the code that much. We better wait for a fix from polyfill.io